templates/controllers/wizard/fileUpload/form/submissionFileMetadataForm.tpl

```diff
@@ -29,19 +29,6 @@
 		{/fbvFormSection}
 	{/fbvFormArea}
 
-	{* Read-only meta-data *}
```

Comment:
I'm not sure we want to completely remove this. For example, it's apparently necessary in publications with heavy use of high-resolution figures (think medical imaging). How do you feel about hiding it during submission, but presenting it in later stages? That would mirror our decision to permit suppression of extended metadata fields during submission but allowing them to appear later for more comprehensive indexing.